src/frontend/komponenter/Oppgavebenk/FilterSkjema.tsx

```diff
@@ -33,9 +35,14 @@ const FilterSkjema: React.FunctionComponent = () => {
                                         key={oppgaveFelt.nøkkel}
                                         id={oppgaveFelt.nøkkel}
                                         label={oppgaveFelt.label}
-                                        onChange={(dato: string | undefined) =>
-                                            settVerdiPåOppgaveFelt(oppgaveFelt, dato ? dato : '')
-                                        }
+                                        onChange={(dato: string | undefined) => {
+                                            settVerdiPåOppgaveFelt(
+                                                oppgaveFelt,
+                                                dato && dayjs(dato, datoformat.ISO_DAG).isValid()
```

Comment:
I think we are using familieDayJs instead of dayJs :) it is the same, but configured!

Reply:
Yes, I saw it. Unfortunately that does not work because it cannot handle invalid input and throws an error, which makes isValid() check ignored

Reply:
What is it in this context? We use familieDayjs with the datepicker in vilkårsvurdering. What is the core problem here? If it is the mapping we could format the date before calling the backend perhaps? :)

Reply:
The problem is saksbehandler can type in date instead of picking from the calendar. In this case if the date format is not ddmmåå, the argument dato of onChange will be the plan string that the saksbehandler typed. The illegal date should not be sent to backend so error handling is needed here. The simplest error handling is what I implemented: clear out the input if the date is invalid. If you want advanced error handling, like support different format of date, we need to discuss, and the solution may not be very straightforward (I have tried some approaches)